addons/product/models/product_template.py

```diff
@@ -538,7 +538,37 @@ def copy(self, default=None):
             default = {}
         if 'name' not in default:
             default['name'] = _("%s (copy)", self.name)
-        return super(ProductTemplate, self).copy(default=default)
+
+        res = super(ProductTemplate, self).copy(default=default)
+
+        # Since we don't copy the attribute line directly, we need to match the extra price and
+        # exclusion of the newly created attribute line values with the old ones.
+        # This matching expects this extra price/exclusion to be uniquely determined by the attribute value e.g. if
+        # two lines uses the same attribute value, the copied extra price will be the same on both lines.
```

Comment:
Could you keep the line length at 100 chars max? =]

Reply:
Done